buildchain/buildchain/docker_command.py

```diff
@@ -71,7 +72,21 @@ def decorated_task(*args: Any, **kwargs: Any) -> Optional[TaskError]:
             try:
                 task_func(*args, **kwargs)
             except expected_exn as err:
-                return TaskError(err)
+                # Output container build log on BuildError
+                if isinstance(err, BuildError):
+                    output_lines = [
+                        item['stream']
+                        if 'stream' in item else item['error']
+                        for item in err.build_log
+                    ]
+                    log = ''.join(output_lines)
+                    err_msg = '{}. Build log:\n{}'.format(str(err), log)
```

Comment:
Why not `err_msg = '{}:\n{}'.format(str(err), log)` to be consistent with `ContainerError`?
Or use sth like `err_msg = '{}. Exec error:\n{}'.format(str(err), err.stderr)` for `ContainerError`